delivery_recipe/digital_service_standards.md

```diff
@@ -2,10 +2,17 @@
 
 As part of driving quality through our delivery process we incorporate the following Digital Service Standards:
```

Comment:
What is meant by Digital Service Standards? That's not referring to the gov DSS is it?